service/src/main/java/org/databiosphere/workspacedataservice/dataimport/tdr/TdrManifestQuartzJob.java

```diff
@@ -71,6 +74,8 @@ public class TdrManifestQuartzJob extends QuartzJob {
   private final RecordSourceFactory recordSourceFactory;
   private final SnapshotSupportFactory snapshotSupportFactory;
   private final SamDao samDao;
+  private final DataImportProperties dataImportProperties;
+  private static final String RESOURCE_TYPE_DATASNAPSHOT = "datasnapshot";
```

Comment:
Since it's Sam-related, is there a more appropriate place for this constant than the TDR import job?

Reply:
Also, another naming suggestion
```suggestion
  private static final String TDR_SNAPSHOT_SAM_RESOURCE_TYPE = "datasnapshot";
```

Reply:
`SamAuthorizationDao` already has `String RESOURCE_NAME_WORKSPACE = "workspace";`, I'd suggest moving this over there and have both resource names in the same place

Reply:
Renamed to TDR_SNAPSHOT_RESOURCE_TYPE